### PR TITLE
tp-qemu: get target monitor by protocol

### PR DESCRIPTION
**Pull request by xutian**
Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

**Comment by xutian** [comment]
ID: 1162033

File: qemu/tests/qmp_event_notification.py [hunk 1]
```diff
@@ -24,10 +24,12 @@ def run(test, params, env):
     vm.verify_alive()
 
     session = vm.wait_for_login(timeout=int(params.get("login_timeout", 360)))
+    qmp_monitor = filter(lambda x: x.protocol == "qmp", vm.monitors)[0]
```
Comment (xutian) [inline qemu/tests/qmp_event_notification.py]: dsfsfasfjsakdf

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by spcui** [comment]
LGTM
Acked-by: @spcui

**Comment by ypu** [comment]
Thanks @xutian @suqinhuang and @spcui will merge it.

**Pull request merged**